packages/charts/src/charts/table/table.styles.ts

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { hexRgb } from './table.utils';
+
+export const TableContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+`;
+
+export const StyledTable = styled.div`
+  border-collapse: collapse;
+  cursor: pointer;
+  display: table;
+  margin: 0;
+`;
+
+export const Header = styled.div<{
+  color: string;
+}>`
+  display: table-row;
+
+  & .sticky {
+    position: -webkit-sticky;
+    position: sticky;
```

Comment:
Is `position: sticky` supported in any of IE version ?

Reply:
`position: sticky` is supported from Edge v16, in older versions it will behave like default: `position: static`